dsc/workflows/sccs.py

```diff
@@ -22,3 +24,17 @@ def s3_bucket(self) -> str:
     @property
     def output_queue(self) -> str:
         return "awaiting AWS infrastructure"
+
+    @staticmethod
+    def get_item_identifier(item_metadata: dict[str, Any]) -> str:
+        """Get 'item_identifier' from item metadata entry.
+
+        For SCCS deposits, this method expects at least one column
+        labeled "item_identifier" or "filename". If both of these
+        columns are present, the column labeled "item_identifier"
+        is selected.
```

Comment:
I think this is good.  While it suggests some loose-ish requirements on the input metadata we ask of stakeholders, it nonetheless explains how it's opinionated here.  Given the small footprint of the `SCSS` class, it really jumps out as a critical part of the class.